.github/workflows/pages-deploy.yml.hook

```diff
@@ -44,7 +44,7 @@ jobs:
         run: |
           baseurl="$(grep '^baseurl:' _config.yml | sed "s/.*: *//;s/['\"]//g;s/#.*//")"
           if [[ -n $baseurl ]]; then
-            echo "SPEC_TEST=_site_no_baseurl" >> $GITHUB_ENV
+            echo "SPEC_TEST=_site$baseurl" >> $GITHUB_ENV
```

Comment:
Thank you very much for your improvement. We can make this PR more perfect:

1. we can just store the `baseurl` to `GITHUB_ENV`:

```diff
- echo "SPEC_TEST=_site$baseurl" >> $GITHUB_ENV
+ echo "BASE_URL=$baseurl" >> $GITHUB_ENV
```